e2e/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  trailingComma: 'none',
+  singleQuote: true,
```

Comment:
it looks like this used to be set to `all`, is that still something we want?

Reply:
In my opinion, no trailing comma is cleaner. Either way, no functional effect.